openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx

```diff
@@ -284,7 +289,8 @@ export const getEntityIcon = (
       break;
   }
 
-  return Icon ? <Icon className={className} style={style} /> : <></>;
+  // If icon is not found, return null
+  return Icon ? <Icon className={className} style={style} /> : null;
```

Comment:
@aniketkatkar97 any reason to return null here?

Reply:
@chirag-madlani I changed this to null so that we can check if the icon is present for the passed entity name and perform actions according to that outside the function. For example, in the first issue described in the description above, I checked if the icon is present there to render the \<Col>, which removed the empty space where it's not needed